src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -6,7 +6,7 @@
     <SkipBuild Condition="'$(RuntimeFlavor)' == 'Mono'">true</SkipBuild>
     <PlatformPackageType>RuntimePack</PlatformPackageType>
     <SharedFrameworkName>$(SharedFrameworkName).Crossgen2</SharedFrameworkName>
-    <OverridePackageId>$(SharedFrameworkName).$(RuntimeIdentifier)</OverridePackageId>
```

Comment:
DOn't we still need the `<OverridePackageId>$(SharedFrameworkName).$(RuntimeIdentifier)</OverridePackageId>` entry here?

Reply:
Lemme check exactly what was going on here, I thought there was some unnecessary complexity, but maybe not.

Reply:
Yup, you were right -- the package id override is still necessary. I confused the zip for the nuget package.